libs/api-sdk/src/groups.ts

```diff
@@ -97,6 +97,34 @@ export async function addMemberByApiKey(
     await request(requestUrl, newConfig)
 }
 
+/**
+ * Adds members to a group using an API Key.
+ * @param groupId Group id.
```

Comment:
`config` object is worth documenting!

Reply:
Yes, true. That's the same Axios request config object. We could say `@param config Axios Request config.` instead of `@param config Request config.`
https://axios-http.com/docs/req_config

What do you think?

Reply:
Great! Yes, it makes sense, thanks! I just created an issue for it: https://github.com/privacy-scaling-explorations/bandada/issues/320